h2o-dist/index.html

```diff
@@ -367,42 +370,56 @@
+    <link rel="stylesheet" href="https://use.typekit.net/gbq5zcz.css">
   </head>
-  <body>
+  <body cz-shortcut-listen="true">
 
     <!--[if lt IE 8]>
       <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
     <![endif]-->
     <div id="container">
-      <div id="title">
-        <p class="links">
-          <a href="http://h2o.ai">H<sub>2</sub>O.ai</a> | <a href="https://github.com/h2oai/h2o-dev">Github</a>
-        </p>
-        <h1><span>H<sub>2</sub>O</span></h1>
-        <p class="version">Version SUBST_PROJECT_VERSION</p>
+      <div id="header">
+          <a href="http://h2o.ai/" id="logo">
+            <img src="https://www.h2o.ai/wp-content/themes/h2o2018/templates/dist/images/h2o_logo.svg"/>
+          </a>
+          <p class="links">
+            <a href="http://h2o.ai/">H2O.ai</a> | <a href="https://github.com/h2oai/h2o-dev">Github</a>
+          </p>
       </div>
-      <div id="subtitle">
-        <p>Fast Scalable Machine Learning API<br/>For Smarter Applications</p>
+      <div id="title">
+        <div class="bg-img">
+          <img src="static/assets/banner.png"/>
+        </div>
+        <div id="banner-content">
+          <h1>H2O</h1>
+          <p class="version">Version SUBST_PROJECT_VERSION</p>
+          <div id="subtitle">
+            <p>Fast Scalable Machine Learning API<br>For Smarter Applications</p>
+          </div>
+        </div>
       </div>
       <div id="content">
-        <div id="tabs">
-          &nbsp; <span id="quickstart-zip-tab" class="active">Download and Run</span>
-          <span id = "quickstart-r-tab">Install in R</span>
-          <span id = "quickstart-py-tab">Install in Python</span>
-          <span id = "quickstart-hadoop-tab">Install on Hadoop</span>
-          <span id = "quickstart-maven-tab">Use from Maven</span>
+        <div id="tabs" class="latest-release">
+          <span id="quickstart-zip-tab" class="active">Download and Run</span>
+          <span id="quickstart-r-tab">Install in R</span>
+          <span id="quickstart-py-tab">Install in Python</span>
+          <span id="quickstart-hadoop-tab">Install on Hadoop</span>
+          <span id="quickstart-maven-tab">Use from Maven</span>
         </div>
-        <!-- Clipboard Icon !-->
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/octicons/2.4.1/octicons.min.css">
-        <script type="text/javascript" src="ZeroClipboard.min.js"></script>
-        <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.0.0-alpha1/jquery.min.js"></script>
-        <script src="//app-ab12.marketo.com/js/forms2/js/forms2.min.js"></script>
+        <script type="text/javascript" src="./Download H2O 3.23.0.4496_files/ZeroClipboard.min.js"></script>
+        <script type="text/javascript" src="./Download H2O 3.23.0.4496_files/jquery.min.js"></script>
```

Comment:
Should be ./static

Reply:
For this page though, that's not a good use model.

Think of this page as a standalone free-standing thing that still needs to work 3 years from now even if the dependencies vanish.  Anything you use should either come from inside this git repo or in the worst case, should live in an h2oai S3 bucket as a static artifact that will never be deleted.

This is not a typical marketing website that works one day and is gone the next.  This page needs to work forever.  Simpler is better, and deleting external dependencies is better.

Reply:
I see. I've added them to ./static